README.md

```diff
@@ -67,6 +72,16 @@ For `batch_size` 1, these are the latency results:
 | TensorRT          fp16 |  1.68       |  8.1          |
 | AITemplate        fp16 |  1.38       |  4.83         |
 
+#### T4 GPU
+Note that `AITemplate` have not supported on T4 GPU yet.
```

Comment:
@Toan-Do 
Change this to:
> Note: `AITemplate` might not support T4 GPU yet. [Check support here](https://github.com/facebookincubator/AITemplate#installation)

Reply:
Markdown:
`> Note: `AITemplate` might not support T4 GPU yet. [Check support here](https://github.com/facebookincubator/AITemplate#installation)`